keyboards/cannonkeys/devastatingTKL/devastatingTKL.c

```diff
@@ -0,0 +1 @@
+#include "devastatingTKL.h"
```

Comment:
```suggestion
#include "devastatingtkl.h"
```

The keyboard directory and <keyboard>.[ch] should be all lowercase.